src/OagBundle/Resources/public/stylesheets/main.css

```diff
@@ -1,39 +1,58 @@
-/* line 4, ../sass/main.scss */
```

Comment:
What was the verdict on ignoring this file?

Reply:
I'll put it in gitignore.

Reply:
Cool - you might need to `git rm` it so it's not tracked anymore if you haven't already